packages/portfolio-contract/test/portfolio.flows.test.ts

```diff
@@ -710,3 +718,52 @@ test.failing(
     }),
   ],
 );
+
+test('open portfolio and claim with Aave position', async t => {
```

Comment:
please add a test for what happens if you put `claim: true` on a withdraw from a USDN position.

I expect it to throw.